deployments/charts/kuma/templates/cni-rbac.yaml

```diff
@@ -21,10 +21,16 @@ metadata:
 rules:
   - apiGroups: [""]
     resources:
-      - pods
       - nodes
     verbs:
       - get
+  - apiGroups: [""]
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
I think I'd add list and watch for nodes too for simplicity, but up to you

Reply:
I'm a fan of least privileges approach, and as merbridge currently doesn't need these permissions I would like to leave it as is

Reply:
fair enough